demo/index.html

```diff
@@ -100,7 +107,16 @@
                       value: false
                     },
                     selected: String,
-                    steps: Array
+                    steps: Array,
+                    i18n: {
+                      type: Object,
+                      value() {
+                        return {
+                          'OPTIONAL': 'Optional',
+                          'step-3-key': 'Preview'
```

Comment:
Oh interesting. So, even for client-specific desires we're going to need to go into each element supporting i18n and individually modify the key-value pairs?